forte/data/data_store.py

```diff
@@ -372,6 +378,31 @@ def _new_annotation(
 
         return entry
 
+    def _new_audio_annotation(
+        self, type_name: str, begin: int, end: int, tid: Optional[int] = None
+    ) -> List:
+        r"""This function generates a new audio annotation with default fields.
+        All default fields are filled with None.
```

Comment:
Actually not all fields are initialized with `None`.Refer to https://github.com/asyml/forte/blob/e2ce0a4684c884068f88d4706184304cddedb1f7/forte/data/data_store.py#L325 for details.

Reply:
I just need to remove the docstring `All default fields are filled with None.` right?

Reply:
yes